src/components/ItemCard.tsx

```diff
@@ -0,0 +1,121 @@
+import { Avatar, Box, Button, Card, CardActions, CardContent, CardMedia, Typography } from "@mui/material";
+import { blueGrey } from "@mui/material/colors";
+import { Album, Groups } from "@mui/icons-material";
+
+const pastelColors = [
```

Comment:
This will be removed. It was temporary for linear background generation, but this was computationally heavy